functions/http/http-form-data/pom.xml

```diff
@@ -148,31 +132,6 @@
           <trimStackTrace>false</trimStackTrace>
         </configuration>
       </plugin>
-      <plugin> <!-- Required for Java 8 (Alpha) functions in the inline editor -->
```

Comment:
Ditto - file an issue if it doesn't break tests?

(Can be all in the same issue, or in different ones - as long as I know to come back and fix it. 🙂)